draft-ietf-quic-applicability.md

```diff
@@ -565,11 +565,19 @@ RESET_STREAM frames, which contain only an error code.
 
 # ACK-only packets on constrained links
 
+The stragegy on sending acknowlegdements in QUIC version 1 is adopted from
+TCP, meaning that usually an QUIC implementation is expected to send one
+packet containing an ACK frame for each two packets received.
 The cost of sending acknowledgments - in processing cost or link
 utilization - could be a significant proportion of available resources if
-these resources are constrained. Reducing the rate at which acknowledgments
-are generated can preserve these resources and improve overall performance,
-for both network processing as well as application-relevant metrics.
+these resources are constrained.  "ACK-only" packets are not reliably
+detectable in QUIC, so network ACK thinning done for TCP is not suitable
+for QUIC traffic. Instead QUIC endpoint need to control the ACK rate
+appropriately and some implenentation may even provide an interface to
```

Comment:
/some implementation may even provide an interface to impact/implementations provide an interface that can reduce/ ??? ... although I like MT's comment also which avoids this.